etc/start_faux

```diff
@@ -2,8 +2,8 @@
 
 echo Running faux $*
 
-umount /etc/resolv.conf # Necessary to run dhclient in docker containers
-
+# Necessary to run dhclient in docker containers
+umount /etc/resolv.conf 
```

Comment:
now you lost the blank space after - it's a separate logical construct from the things that follow it

Reply:
changed